plenum/test/view_change/test_elections_after_view_change.py

```diff
@@ -1,40 +0,0 @@
-from functools import partial
-
-from stp_core.loop.eventually import eventually
-from stp_core.loop.looper import Looper
-from plenum.test import waits
-from plenum.test.delayers import ppDelay
-from plenum.test.helper import sendReqsToNodesAndVerifySuffReplies
-from plenum.test.test_node import TestNodeSet, getNonPrimaryReplicas, \
-    checkProtocolInstanceSetup, checkViewChangeInitiatedForNode
-
-nodeCount = 7
-
-
-# noinspection PyIncorrectDocstring
-def testElectionsAfterViewChange(delayed_perf_chk, looper: Looper,
```

Comment:
A good way to write tests is to test each feature independently and incrementally.